src/Lean/Linter/Basic.lean

```diff
@@ -66,7 +66,7 @@ def unusedVariables : Linter := fun stx => do
         match expr with
         | .fvar id _ => modify (·.insert id)
         | _          => pure ()
-        return false)
+        return true)
```

Comment:
Am I misunderstanding something or does this mean that the docstring of `Expr.forEach'` is wrong?

``[...] If `f t` return true, then t's children are not visited.``

Reply:
Fixed, thanks!